src/products/android-l10n/review_android_strings.md

```diff
@@ -1,37 +1,77 @@
+
+If automation fails to import exceptions from the XML files, you can manually add an exception to the  [linter\_config.json](https://github.com/mozilla-l10n/android-l10n/blob/main/.github/scripts/linter_config.json) file. To do this, include the string ID displayed in the error log.
+
+In case of issues, you can comment and CC the developer who introduced the strings. This can be done directly in the corresponding `mozilla-l10n` GitHub PR.
+
+These strings will be available for localizers in Pontoon only after the PR is merged, so it’s important to communicate promptly with developers if any fixes are needed.
 
 ## Wrapping up your work
 
-So you’ve reviewed the strings during the cycle, and the PR looks good - this means you can now merge the PR so the strings get exposed in Pontoon.
+So you’ve reviewed the strings during the cycle, and the PR looks good — this means you can now merge the PR so the strings get exposed in Pontoon.
 
-Pontoon ([here for Firefox for Android](https://pontoon.mozilla.org/projects/firefox-for-android/) and [here for Focus for Android](https://pontoon.mozilla.org/projects/focus-for-android/)) should get the new strings automatically after a few minutes, but it’s always a good idea to double check just in case.
+Pontoon ([here for Firefox for Android](https://pontoon.mozilla.org/projects/firefox-for-android/) and [here for Focus for Android](https://pontoon.mozilla.org/projects/focus-for-android/)) should get the new strings automatically after a few minutes, but it’s always a good idea to double-check just in case. Note that `android-components` files are included and localized within the Firefox for Android project on Pontoon.
 
-One last thing is to not forget to update the l10n completion deadline under Pontoon resources for each product.
+Remember to update the localization completion deadline under Pontoon resources for each project. For each release, the deadline falls one day before the “Release Candidate” date of the current [Nightly version](https://whattrainisitnow.com/release/?version=nightly). Since android-l10n products follow the same four-week release cycle as other Firefox browsers, these deadlines must be updated regularly. To update, navigate to the [Pontoon admin](https://pontoon.mozilla.org/admin/) page, select your project, and update the “Target date”.
```

Comment:
This is wrong, the RC date relevant for the deadline is the one of the version currently in Beta, not Nightly.

Reply:
Right - I made that mistake since I look at the Nightly page on the release website